server/src/test/java/org/elasticsearch/transport/ConnectionManagerTests.java

```diff
@@ -124,11 +124,10 @@ public void onNodeDisconnected(DiscoveryNode node, Transport.Connection connecti
         assertEquals(1, nodeDisconnectedCount.get());
     }
 
-    @AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/49903")
     public void testConcurrentConnectsAndDisconnects() throws BrokenBarrierException, InterruptedException {
```

Comment:
Should we rename this to just `testConcurrentConnects`? It doesn't obviously do any disconnecting.